Gruntfile.js

```diff
@@ -137,7 +141,7 @@ module.exports = function(grunt) {
 
     grunt.registerTask('addPolyfill', function() {
         var fs = require('fs');
-        var polyfillPath = './node_modules/grunt-babel/node_modules/babel-core/browser-polyfill.js';
+        var polyfillPath = './node_modules/babel-polyfill/dist/polyfill.js';
```

Comment:
I was really hoping that we could ditch this, but things blowup big time in IE without it 💥  That being said, I'm quite certain that we don't need to include the *entire* polyfill here. We could selectively load it on the app side of things only for browsers that need it, or just load in the specific polyfills that we actually need. Out of scope for this PR, but something we might want to consider since this adds a considerable amount of unnecessary bloat in most environments.